README.md

```diff
@@ -7,9 +7,9 @@
 An abstraction layer for mathematical optimization solvers. Replaces [MathProgBase](https://github.com/JuliaOpt/MathProgBase.jl).
 
 [docs-stable-img]: https://img.shields.io/badge/docs-stable-blue.svg
-[docs-latest-img]: https://img.shields.io/badge/docs-latest-blue.svg
+[docs-dev-img]: https://img.shields.io/badge/docs-dev-blue.svg
 [docs-stable-url]: http://www.juliaopt.org/MathOptInterface.jl/stable
-[docs-latest-url]: http://www.juliaopt.org/MathOptInterface.jl/latest
+[docs-dev-url]: http://www.juliaopt.org/MathOptInterface.jl/dev
```

Comment:
Will all the links to `latest` be broken now?

Reply:
No they are currently redirected but they should progressively be updated.

> The directories (URLs) that change from version to version (e.g. latest/, vX.Y) are implemented as symlinks on the gh-pages branch.

See https://github.com/JuliaDocs/Documenter.jl/blob/master/CHANGELOG.md